flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -666,6 +666,18 @@ public boolean isForceKryoEnabled() {
         return configuration.get(PipelineOptions.FORCE_KRYO);
     }
 
+    public void enableForceAvroKryo() {
+        setForceAvroKryo(true);
+    }
+
+    private void setForceAvroKryo(boolean forceAvroKryo) {
+        configuration.set(PipelineOptions.FORCE_KRYO_AVRO, forceAvroKryo);
+    }
+
+    public Optional<Boolean> isForceAvroKryoEnable() {
+        return configuration.getOptional(PipelineOptions.FORCE_KRYO_AVRO);
+    }
+
```

Comment:
I am curious why there are three public methods introduced that were not included in FLIP-398.